android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/ResourceExtension.kt

```diff
@@ -382,6 +383,14 @@ fun isValidResourceType(resourceCode: String): Boolean {
   }
 }
 
+fun ImplementationGuide.retrieveImplementationGuideDefinitionResources():
+  List<ImplementationGuide.ImplementationGuideDefinitionResourceComponent> {
+  val resources =
+    mutableListOf<ImplementationGuide.ImplementationGuideDefinitionResourceComponent>()
+  this.definition.resource.forEach { resources.add(it) }
+  return resources
+}
+
```

Comment:
what's the plan for this fun?

Reply:
Was added in an initial iteration of the functionality. Will be reviewed and removed if not needed